### DISCUSSION TITLE
Scale up and down with different metric

### DISCUSSION THREAD

**Discussion by daniele-arosio** [kedacore/keda#3765] [Q&A / Need Help]
Is it possible to use a metric to scale up a pod and another to scale down?

For example, I want to scale up a pod when the number of messages in a Rabbitmq queue is greater than a threshold.
But I want to scale down the pod only when the number of "unacknowledged" messages is 0. In this way, I avoid the case where the pod is scaled down while it has not finished to process the message.

If I use both the metrics to scale up/down (number of messages greater than a threshold or number of "unacknowledged" messages greater than 0) I don't have the desired behaviuor. In fact, in this way, just an "unacknowledged" message will scale up the pod to the maximum number of replicas.

Any suggestion?
Thank you very much.

**Comment by tomkerkhove** [comment]
Not at the moment but that's a good feature request! It might be related to https://github.com/kedacore/keda/issues/2614 though.
